src/CMakeLists.txt

```diff
@@ -167,6 +169,7 @@ target_link_libraries(migraphx PRIVATE -lstdc++fs)
 endif()
 
 target_include_directories(migraphx SYSTEM PUBLIC $<BUILD_INTERFACE:${HALF_INCLUDE_DIR}>)
+target_include_directories(migraphx PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>)
```

Comment:
Actually, I think this line is not needed as `rocm_install_targets` will add the include directory paths.